<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_main_window">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>320</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="majorAlphabet">
         <property name="text">
          <string>Major League Stadiums Sorted in Alphabetic Order</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="majorTeam">
         <property name="text">
          <string>Major League Stadiums Sorted by Team Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="americanTeam">
         <property name="text">
          <string>Amercan League Stadiums Sorted by Team Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nationalTeam">
         <property name="text">
          <string>National League Stadiums Sorted by Team Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="grassSurface">
         <property name="text">
          <string>View Grass Surface Stadiums</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="byDate">
         <property name="text">
          <string>View Stadiums in Chronological Order</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="planTrip">
         <property name="text">
          <string>Plan a Trip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="adminLogin">
         <property name="text">
          <string>Admin Log-in</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quit">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>321</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; text-decoration: underline;&quot;&gt;Take Me Out to the Ballgame&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_view_stadiums">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>291</width>
        <height>321</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>289</width>
         <height>319</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="backToMain">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewInformation">
      <property name="geometry">
       <rect>
        <x>462</x>
        <y>80</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>View Information</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_plan_trip">
     <widget class="QPushButton" name="backToMain2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_admin_login">
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>241</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="userName">
           <property name="text">
            <string>User Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="userNameLine"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="password">
           <property name="text">
            <string>Password  :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordLine"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="login">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Admin Log-in&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_view_information">
     <widget class="QLabel" name="sadiumName">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Stadium Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="streetAddress">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Street Address: </string>
      </property>
     </widget>
     <widget class="QLabel" name="city">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>City: </string>
      </property>
     </widget>
     <widget class="QLabel" name="state">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>State: </string>
      </property>
     </widget>
     <widget class="QLabel" name="zip">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Zip: </string>
      </property>
     </widget>
     <widget class="QLabel" name="boxOffice">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Box Office: </string>
      </property>
     </widget>
     <widget class="QLabel" name="dateOpened">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date Opened: </string>
      </property>
     </widget>
     <widget class="QLabel" name="capacity">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Capacity</string>
      </property>
     </widget>
     <widget class="QLabel" name="stadiumInformationTitle">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Stadium Information</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineStadiumName">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineStreetAddress">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineCity">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineState">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineZip">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineBoxOffice">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineCapacity">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>280</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineDateOpened">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="backToList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineCapacity_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>280</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineDescription1">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>280</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineDescription2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5"/>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
